File: src/main/java/com/my/admin/app/commerce/model/document/UserRole.java
package com.my.admin.app.commerce.model.document;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.io.Serializable;

@Document(collection = "users_roles")
@Data
public class UserRole implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    private Long userId;

    @Id
    private Long roleId;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/document/UserRole.kt
package com.bk.reactive.app.commerce.admin.my.model.document

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "users_roles")
data class UserRole(
    @Id
    val id: String,
    val userId: Long,
    val roleId: Long
)